app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC, abstractmethod
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, favourite_dish: int) -> str:
+        self.favourite_dish = favourite_dish
```

Comment:
![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/58142395/c51ac233-77d1-4e36-9c7c-893c8937b910)